engine/execution/ingestion/engine.go

```diff
@@ -424,6 +431,13 @@ func (e *Engine) BlockProcessable(b *flow.Header) {
 		return
 	}
 
+	// skips blocks at or above requested stop height
+	stopHeight := e.stopAtHeight.Load()
+	if stopHeight > 0 && b.Height >= stopHeight {
```

Comment:
q: does this equal to checking `pauseExecution`? if so, should it precede the previous "pauseExecution"?

Reply:
checking pauseExecution is faster so left it as a first check

Reply:
so do we still see the warning msg below when `stopHeight > 0 && b.Height >= stopHeight` is true?

Reply:
yes, until the stopHeight gets finalized